openedx_learning/core/publishing/api.py

```diff
@@ -91,7 +91,7 @@ def create_publishable_entity_version(
             version_num=version_num,
             title=title,
             created=created,
-            created_by=created_by,
+            created_by_id=created_by,
```

Comment:
What's the reason for this change?  What is it fixing?

Reply:
It's solving the following error:

The method takes `created_by` as an integer id of the user while we are passing it as a User object in the `PublishableEntityVersion.objects.create(` function.

<img width="1237" alt="Screenshot 2023-09-27 at 9 37 07 AM" src="https://github.com/openedx/openedx-learning/assets/25842457/8b9030ba-92c5-4e39-bf3d-861df07b4b25">

Code on the same lines in the same file:
https://github.com/openedx/openedx-learning/blob/main/openedx_learning/core/publishing/api.py#L58-L71